services/bls_aggregation/blsagg.go

```diff
@@ -301,6 +305,14 @@ func (a *BlsAggregatorService) singleTaskAggregatorGoroutineFunc(
 					TotalStakeIndices:            indices.TotalStakeIndices,
 					NonSignerStakeIndices:        indices.NonSignerStakeIndices,
 				}
+
+				blsAggregationServiceResponse.NonSignersPubkeysG1,
+					blsAggregationServiceResponse.NonSignerQuorumBitmapIndices =
+					sortByOprId(
+						blsAggregationServiceResponse.NonSignersPubkeysG1,
+						blsAggregationServiceResponse.NonSignerQuorumBitmapIndices,
+					)
+
```

Comment:
I think that this needs to be moved above, before the call to GetCheckSignaturesIndices, otherwise the NonSignerStakeIndices returned won't be in the correct order?

If this worked for you, I'm a bit puzzled. Am I missing something here?

Reply:
refined